db/data/test-data/index.js

```diff
@@ -1,4 +1,11 @@
-exports.articleData = require('./articles.js');
-exports.commentData = require('./comments.js');
-exports.topicData = require('./topics.js');
-exports.userData = require('./users.js');
+exports.articleData = require("./articles.js");
+exports.commentData = require("./comments.js");
+exports.topicData = require("./topics.js");
+exports.userData = require("./users.js");
+
+exports.indexTest = {
```

Comment:
You have essentially duplicated what is already happening in this file - these changes should be removed and the file should be put back to how it looked originally.